modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/frontend/util/CommerceOrderClayTableUtil.java

```diff
@@ -61,6 +63,9 @@
  */
 public class CommerceOrderClayTableUtil {
 
+	public static final String DATE_AND_TIME_FORMAT =
+		"MMM dd, yyyy hh:mm:ss aa";
+
```

Comment:
@CrescenzoRegaLF can we please replace this with

_commerceOrderDateFormatDate = FastDateFormatFactoryUtil.getDate(
   DateFormat.MEDIUM, themeDisplay.getLocale(),
   themeDisplay.getTimeZone());
_commerceOrderDateFormatTime = FastDateFormatFactoryUtil.getTime(
   DateFormat.MEDIUM, themeDisplay.getLocale(),
   themeDisplay.getTimeZone());